lib/paysafe/account_management/look_up_merchant_account.rb

```diff
@@ -0,0 +1,21 @@
+module Paysafe
+  module AccountManagement
+    class LookUpMerchantAccount < JsonObject
+      attr_accessor :id
```

Comment:
Is this the same order Paysafe docs provide?

Reply:
it is not depend on doc